hs_discover/src/components/Search.vue

```diff
@@ -347,7 +369,7 @@ export default {
     if (this.subjectFilter) {
       const escaped = [];
       this.subjectFilter.forEach((x) => {
-        escaped.push(x.replaceAll('\\', '\\\\'));
+        escaped.push(x.replace(/\\/g, '\\\\'));
```

Comment:
Nice to know the regex in the first parameter of the replace() does not need to be quoted as a string.